build.sh

```diff
@@ -335,12 +335,13 @@ download_sync() {
     TMP_DIR="$TMPDIR/sync_bin"
     mkdir -p "${TMP_DIR}"
     SYNC_TMP_TAR="${TMP_DIR}/sync-${REALM_SYNC_VERSION}.tar.xz.tmp"
+    rm -rf "$SYNC_TMP_TAR" # ensure we don't have a corrupt download
```

Comment:
This isn't a change that adds caching, it is a change to delete a potentially bad version that is already there. So we already had it "cached", this just avoids a problem I ran into during testing.